src/Sulu/Bundle/WebsiteBundle/Controller/ExceptionController.php

```diff
@@ -60,6 +62,8 @@ public function __construct(
         Environment $twig,
         $debug
     ) {
+        @trigger_error(__CLASS__ . ' is deprecated since version sulu/sulu 2.0 and will be removed in 3.0. Use the ErrorController instead.', E_USER_DEPRECATED);
```

Comment:
Isn't that fully in our control? I guess we don't have this registered for a route anymore, right? Then I would argue we could simply remove that controller, can't we?

Reply:
Aslong as he did not overwrite it and extend from it and configure its own twig exception controller we could remove it.

Reply:
Okay, lets keep it, it shouldn't hurt too much, I guess.